spec/spec_helper_acceptance.rb

```diff
@@ -164,7 +163,6 @@ def teardown_puppet_on(host)
 package { ['puppet-agent', 'puppet', 'mcollective', 'mcollective-client']: ensure => purged }
   EOS
   on host, puppet('apply', '-e', "\"#{pp}\"")
-  remove_defaults_on host
```

Comment:
OK. Since there's a beaker ticket can you up date this with a quick comment to that effect (or move the reomoval to its own commit that mentions the ticket) so we have some info in this repo about what we're blocked on?

Once that's done I'll be happy to click the button here